### ISSUE TITLE
Cocos2dx doesn't works well on device(lumia 920)

### ISSUE THREAD

**Issue by sandcu** [cocos2d-x/cocos2dx-win8#20]
I tried the latest version and found that
the hello world project does well on the Emulator but not on the  device
I tried the demo on Nokia lumia 920 and HTC 8x,it only drawed on the left bottom side of screen 
![wp_ss_20130122_0004](https://f.cloud.github.com/assets/530010/85478/3471b8a0-645e-11e2-930d-8ebd57cefb72.png)
![wp_ss_20130122_0003](https://f.cloud.github.com/assets/530010/85479/3bb9bcfc-645e-11e2-89ec-d0e1464b9546.png)
![wp_ss_20130122_0001](https://f.cloud.github.com/assets/530010/85480/3dc4302c-645e-11e2-9954-4cddec35cc43.png)
![wp_ss_20130122_0002](https://f.cloud.github.com/assets/530010/85481/3f75ee7e-645e-11e2-81b4-68281a969202.png)

**Comment by lihex** [comment]
PLZ,try to set the design resolution in HelloWorld\Classes\AppDelegate.cpp
53line mainView->setDesignResolution(height, with);

**Comment by sandcu** [comment]
yep,I've tried this but still got some other problems,I dicussed this with other guys about this at http://www.developer.nokia.com/Community/Wiki/Creating_a_New_Cocos2d-x_Project_for_Windows_Phone_8
